docs/analytics/screen-tracking.md

```diff
@@ -29,6 +29,7 @@ import { NavigationContainer } from '@react-navigation/native';
     if (previousRouteName !== currentRouteName) {
       await analytics().logScreenView({
         screen_name: currentRouteName,
+        screen_class: currentRouteName
```

Comment:
Same as the other one - we default to trailing commas now. Super tiny nitpick, I know

```suggestion
        screen_class: currentRouteName,
```